drivers/bridge/bridge.go

```diff
@@ -674,6 +675,7 @@ func (d *driver) createNetwork(config *networkConfiguration) error {
 	bridgeAlreadyExists := bridgeIface.exists()
 	if !bridgeAlreadyExists {
 		bridgeSetup.queueStep(setupDevice)
+		config.IfaceManaged = true
```

Comment:
On daemon reload, this will cause all existing networks' bridges to be kept as non-managed.
Somehow we need to make sure the existing networks in store are marked as managed, but after this feature they should not.

Maybe we can do this by adding a different flag instead, which is true only for network for which `com.docker.network.bridge.name` was specified at creation time. And the flag needs to be saved to store.

WDYT ?